### PR TITLE
Add content/pt/docs/tutorials/kubernetes-basics/create-cluster

### PR DESCRIPTION
**Pull request by femrtnz**
related to #13939

### CODE REVIEW

**Comment by femrtnz** [comment]
Hey @jhonmike can you review this one?

thanks!

File: content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+        <div class="katacoda">
+            <div class="katacoda__alert">
+                Esta tela é muito estreita para interagir com o Terminal, use um desktop / tablet.
+            </div>
+            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/1" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;"></div>
+        </div>
+        <div class="row">
+            <div class="col-md-12">
+                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">Continue to Module 2<span class="btn__next">›</span></a>
```
Comment (jhonmike) [inline content/pt/docs/tutorials/kubernetes-basics/create-cluster/cluster-interactive.html:28]:
```suggestion
                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/" role="button">Continue para o Módulo 2<span class="btn__next">›</span></a>
```

**Review by jhonmike — COMMENTED** [review]
I found only one excerpt without translation, congratulations on the translation 👏

**Review by jhonmike — APPROVED** [review]
/lgtm
/approve

**Pull request merged**